bridge/client/app/_services/data.service.mock.ts

```diff
@@ -46,6 +47,16 @@ export class DataServiceMock extends DataService {
     return of({});
   }
 
+  public projectExists(projectName: string): Observable<boolean | undefined> {
```

Comment:
There is no difference between this function and the one in the DataService, therefore we don't need to overwrite it here.